halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/ClouddriverBootstrapProfileFactory.java

```diff
@@ -62,16 +69,26 @@ protected void setProfile(Profile profile, DeploymentConfiguration deploymentCon
 
     account.makeBootstrappingAccount(artifactSourcesConfig);
 
-    NodeIterator children = providers.getChildren();
-    Provider child = (Provider) children.getNext();
-    while (child != null) {
-      boolean isBootstrappingAccount = child.getAccounts().stream().anyMatch(a -> ((Account) a).getName().equals(account.getName()));
+    NodeIterator providerNodes = providers.getChildren();
+    Provider provider;
+    while ((provider = (Provider) providerNodes.getNext()) != null) {
+      boolean isBootstrappingProvider = provider.getAccounts()
+                                                .stream()
+                                                .anyMatch(a -> ((Account) a).getName().equals(account.getName()));
       // ugly check for docker registry here since it can be depended on by the bootstrapping account, meaning it's not safe to disable
-      if (child.providerType() != Provider.ProviderType.DOCKERREGISTRY && !isBootstrappingAccount) {
-        child.setEnabled(false);
+      if (provider.providerType() != Provider.ProviderType.DOCKERREGISTRY && !isBootstrappingProvider) {
+        provider.setEnabled(false);
       }
 
-      child = (Provider) children.getNext();
+      if (isBootstrappingProvider) {
+        List bootstrapAccounts = (List) provider.getAccounts()
+                                                .stream()
```

Comment:
The rest of the project aligns these call chains two indent levels rather than against the first method call.